cmake/EthCompilerSettings.cmake

```diff
@@ -33,7 +33,7 @@ if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR ("${CMAKE_CXX_COMPILER_ID}" MA
 	# to fix warnings as they arise, so they don't accumulate "to be fixed later".
 	add_compile_options(-Wall)
 	add_compile_options(-Wextra)
-	add_compile_options(-Werror)
+	# add_compile_options(-Werror)
```

Comment:
This flag is there to turn all warnings into errors.  For some reason, this always throws on my setup (even vanilla/upstream).  @K-Ho if you haven't run into this we can revert and I'll just keep making the change locally.